whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTestIntegration.java

```diff
@@ -2007,6 +2007,27 @@ public void search_route_roa_origin_and_prefix_length_mismatch_validation_enable
                 "either remove this route: object or update or delete the ROA.\n"));
     }
 
+    @Test
+    public void search_route_one_invalid_one_valid_roas_makes_it_valid() {
+        databaseHelper.addObject(RpslObject.parse("" +
+                "route:           92.38.45.0/24\n" +
```

Comment:
Where is the valid ROA?

I can only find the ROA with prefix 92.142.0.0/16 and ASN AS3215 in roas.json.

Reply:
is exactly same example that Ricardo gave to us
I did debug and I have two 

    { "asn": "AS44546", "prefix": "92.38.0.0/17", "maxLength": 24, "ta": "arin" },
    { "asn": "AS61979", "prefix": "92.38.44.0/22", "maxLength": 24, "ta": "ripe" }

Reply:
OK, I see the extra ROAs in the branch (diff)